src/content/h1/tags/readme.md

```diff
@@ -46,4 +48,20 @@ Invalid tags can often be corrected by resetting fields and re-saving the tag us
 ## Padding
 Fields are not always densely packed within a tag; sometimes there exists unused space between them without any known purpose. These spaces are often referred to as _padding_. Generally, any sort of data could be stored in these spaces without affecting the tags, and some community tools use this space to retain extra metadata.
 
+## Headers
+
+All tags have a common header. This header makes up the first 64 bytes of data, and contains the following fields:
+
+| Field       | Type     | Comments |
+|-------------|----------|----------|
+| padding     | pad      | 36 bytes |
+| tag group   | `uint32` | 4 character tag group ID encoded as an int |
+| checksum    | `uint32` |          |
+| header size | `uint32` | Default: 64 |
+| flags       | `uint64` | A 64 bit bitfield |
+| version     | `uint16` | Default: 1 |
```

Comment:
Not sure how I feel about this being described in the same way that `header size` is, that almost universally is 64 (I think the tag reading code in the official tools is even hardcoded to assume that), while `version` iirc is set to whatever the definition version was set to - it just happens to be incremented from 1.